src/Raven.Server/Documents/ETL/Providers/Queue/QueueBrokerConnectionHelper.cs

```diff
@@ -57,8 +56,8 @@ public static void SetupKafkaClientConfig(ClientConfig config, KafkaConnectionSe
     {
         if (settings.UseRavenCertificate && certificateHolder?.ClientCertificate != null)
         {
-            config.SslCertificatePem = ExportAsPem(new PemObject("CERTIFICATE", certificateHolder.ClientCertificate.RawData));
-            config.SslKeyPem = ExportAsPem(certificateHolder.PrivateKey.Key);
+            config.SslCertificatePem = certificateHolder.ClientCertificate.ExportCertificatePem();
+            config.SslKeyPem = (certificateHolder.PrivateKey as RSA).GetExportableRsaPrivateKey().ExportRSAPrivateKeyPem();
```

Comment:
Same note here on `ECDsa` support

Reply:
It is a separate issue, this one is a 1:1 transition